ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplLoadWorkMBean.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.hadoop.hive.ql.exec.repl;
+
+public interface ReplLoadWorkMBean {
+
+  /**
+   * Gets the name of the database which is being loaded.
+   * @return the source database name.
+   */
+  public String getSourceDatabase();
+
+  /**
+   * Gets the name of the database on which load is happening.
+   * @return the target database name.
+   */
+  public String getTargetDatabase();
+
+  /**
+   * Gets the stage of Replication.
+   * @return INCREMENTAL or BOOTSTRAP load.
+   */
+  public String getStage();
+
+  /**
+   * Gets the name of scheduled query being run.
+   * @return the name of the scheduled query.
+   */
+  public String getScheduledQueryName();
```

Comment:
Add current event id processed and last repl id?

Reply:
The repl stats tracks what comes to replLogger, the Last event id is there, I have added that in the JMX

Reply:
what about current event id?

Reply:
added